ocs_ci/ocs/ui/views.py

```diff
@@ -240,10 +244,26 @@
         By.CSS_SELECTOR,
     ),
     "confirm_pvc_deletion": ('button[data-test="confirm-action"]', By.CSS_SELECTOR),
-    "pvc_test": ('a[data-test-id="test-pvc-fs"]', By.CSS_SELECTOR),
     "search_pvc": ('input[data-test-id="item-filter"]', By.CSS_SELECTOR),
 }
 
+pvc_4_7 = {
+    "test-pvc-fs": ('a[data-test-id="test-pvc-fs"]', By.CSS_SELECTOR),
+    "test-pvc-rbd": ("a[title='test-pvc-rbd']", By.CSS_SELECTOR),
+    "Block": ("input[value='Block']", By.CSS_SELECTOR),
+    "Filesystem": ("input[value='Filesystem']", By.CSS_SELECTOR),
+}
+
+pvc_4_8 = {
+    "ReadWriteMany": ("input[value='ReadWriteMany']", By.CSS_SELECTOR),
+    "pvc_actions": ("button[aria-label='Actions']", By.CSS_SELECTOR),
+    "ReadWriteOnce": ("input[value='ReadWriteOnce']", By.CSS_SELECTOR),
+    "Filesystem": ("input[value='Filesystem']", By.CSS_SELECTOR),
```

Comment:
Why duplicate the values that are identical in 4.7 and 4.8?

Reply:
Removed identical locators, please close the requested change so that we can merge the PR.